background_scripts/commands.coffee

```diff
@@ -94,6 +101,24 @@ Commands =
     @keyToCommandRegistry = {}
     @mapKeyToCommand { key, command } for own key, command of defaultKeyMappings
 
+  generateKeyStateMapping: ->
```

Comment:
What this does is actually explained in mode_key_handler.coffee. It would be nice if this function had a docstring either referencing `mode_key_handler.coffee` or explaining what the method does.